### DISCUSSION TITLE
How to install pimcore 6. X

### DISCUSSION THREAD

**Discussion by jelly0930** [pimcore/pimcore#9947] [Q&A / Support]
### Improvement description

Through the official documentation, I tried to install pimcore 6. X using the following command
`COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/demo my-project 6.9.6`
Error message: no pimcore version 6.9.6
Then I tried another command
`COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/pimcore my-project 6.9.6`
Although it can be downloaded, there is no `./vendor/bin/pimcore-install` and no `public` directory
So how to install pimcore 6. X

**Comment by aweichler** [comment]
You can look at the documentation ("https://pimcore.com/docs/pimcore/6.9/Development_Documentation/Getting_Started/Installation.html"), here is everything you need.
If you create a project you need the Version of the installation package and not the version of "pimcore" itself to install Pimcore 6.9.
`COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/demo my-project ~1` 
or
`COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/skeleton my-project ~2`

**Comment by jelly0930** [comment]
I've read  the documentation ("https://pimcore.com/docs/pimcore/6.9/Development_Documentation/Getting_Started/Installation.html")
but Pimcore is still the latest version installed through the 6.9 documentation
Is there any other way？

**Comment by aweichler** [comment]
You are right in the documentation the version for the installation package is missing.
Have you tried the commands i wrote above?
These should work.

**Comment by jelly0930** [comment]
like this？but it's not work
![image](https://user-images.githubusercontent.com/37065209/128299687-ee93cb33-501f-47b6-ac02-8c95fdf19032.png)

**Comment by aweichler** [comment]
I think that is another error, which is not related to pimcore. Can you try to enter the command in the shell "bash"?

**Comment by jelly0930** [comment]
I changed the Ubuntu 20 to download it successfully. But the version is v1.6. Is it too old？

![image](https://user-images.githubusercontent.com/37065209/128300299-d6586d3f-2485-4861-a994-4153e702d541.png)

**Comment by aweichler** [comment]
The version "1.6.24" is the version of the demo package. I think you should have a look at the [composer.json](https://github.com/pimcore/demo/blob/v1.6.24/composer.json) in that package version. You can see that Pimcore 6.9.x is installed.
It is the same with "[pimcore/skeleton](https://github.com/pimcore/skeleton/tree/v2.8.22)". Version 2.8.* is using Pimcore 6.9.

**Comment by jelly0930** [comment]
I got it, thank you for your patient guidance
